ee-9/feature-pack/pom.xml

```diff
@@ -534,6 +535,10 @@
                                 <exclude>org.wildfly:wildfly-clustering-infinispan-client\z</exclude>
                                 <exclude>org.wildfly:wildfly-clustering-infinispan-extension\z</exclude>
                                 <exclude>org.wildfly:wildfly-clustering-web-infinispan\z</exclude>
+                                <!-- Don't transform Messaging modules -->
```

Comment:
A more specific message explaining why is what I'm looking for in these comments, e.g.

'Used for legacy JMS compatibility'